src/store/cesium.ts

```diff
@@ -99,19 +104,34 @@ watch(visibleOverlayIds, () => {
   });
 }, { deep: true });
 
-export const addGeojson = (geojson: GeoJsonShape): Entity => {
-  const cesiumPoints: RGDResult[] = [];
-  geojson.coordinates[0].forEach((e: number[]) => {
-    cesiumPoints.push(Cesium.Cartesian3.fromDegrees(e[0], e[1]));
-  });
-  return cesiumViewer.value.entities.add({
-    polygon: {
-      hierarchy: cesiumPoints,
-      material: new Cesium.ColorMaterialProperty(
-        Cesium.Color.fromRandom({ alpha: 0.5 }),
-      ),
-    },
-  });
+export const addGeojson = (geojson: Polygon | MultiPolygon): Entity[] => {
+  const makeEntity = (
+    coordinates: Position[][],
+    parent: Entity | undefined = undefined,
+  ): Entity => {
+    const cesiumPoints: RGDResult[] = [];
+    coordinates[0].forEach((e: Position) => {
+      cesiumPoints.push(Cesium.Cartesian3.fromDegrees(e[0], e[1]));
+    });
```

Comment:
We can also convert this to use `map` to assign `cesiumPoints`

Reply:
Changed in 73d01e4